http_stubs/templatetags/stub_tags.py

```diff
@@ -16,3 +18,15 @@ def get_absolute_url_tag(context: Dict, url: Url) -> Url:
     :returns: absolute url
     """
     return context.get('request').build_absolute_uri(url)
+
+
+@register.filter(name='headers_to_list')
+@stringfilter
+def get_headers_list(headers: AnyStr, separator: AnyStr) -> List:
+    """Filter that creates a list from a string of headers(dict).
```

Comment:
I'd start from [Python str.split](https://docs.python.org/3/library/stdtypes.html#str.split) description because it is more clear. E.g.:
`Return a list of the lines in the string representation of the request headers, using sep as the delimiter string.`